utils/api.go

```diff
@@ -14,6 +15,24 @@ func GetMessageHttpResponse(statusCode int, msg string) events.APIGatewayProxyRe
 		StatusCode: statusCode,
 	}
 }
+
+func GetDataHttpResponse(statusCode int, msg string, data interface{}) events.APIGatewayV2HTTPResponse {
```

Comment:
Thanks for adding this!
NOTE: Refactoring the API is another task and this code might be changed in the future.

Reply:
Alright